client/schema.go

```diff
@@ -0,0 +1,37 @@
+package client
+
+import (
+	"fmt"
+	"github.com/hashicorp/vault/sdk/helper/strutil"
+)
+
+// SupportedSchema returns a slice of different OpenLDAP schemas supported
+// by the plugin.  This is used to change the FieldRegistry when modifying
+// user passwords.
+func SupportedSchema() []string {
```

Comment:
Why have this be a function?

Reply:
I made `SupportedSchema` a function because you can't declare const slices. I suppose it could be a global level variable.  Do you have a preference?